extensions/kotlin/runtime/pom.xml

```diff
@@ -19,6 +19,20 @@
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-bootstrap-maven-plugin</artifactId>
                 <configuration>
+                    <parentFirstArtifacts>
+                        <parentFirstArtifact>org.jetbrains.kotlin:kotlin-stdlib-jdk8</parentFirstArtifact>
+                        <parentFirstArtifact>org.jetbrains.kotlin:kotlin-stdlib-jdk7</parentFirstArtifact>
```

Comment:
I don't know that we need the `-jdk8` and `-jdk7` deps any more.  The kotlin docs suggest that 1.4 doesn't need those and we can just dep on `stdlib`.